src/v/redpanda/admin_server.cc

```diff
@@ -5445,3 +5449,34 @@ admin_server::sampled_memory_profile_handler(
     co_return co_await ss::make_ready_future<ss::json::json_return_type>(
       std::move(resp));
 }
+
+void admin_server::register_wasm_transform_routes() {
+    register_route_raw_async<user>(
```

Comment:
What is the reasoning to allow regular users to `POST /v1/transform/deploy`? Should this be superuser only?

Reply:
for now yes. We want to start enforcing ACLs here too, but it's probably better to start with superuser.